app/uk/gov/hmrc/trusts/models/RegistrationTrustResponse.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.trusts.models
+
+import play.api.Logger
+import play.api.http.Status._
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.{HttpReads, HttpResponse}
+import uk.gov.hmrc.trusts.exceptions._
+import uk.gov.hmrc.trusts.utils.Contstants._
+
+sealed trait RegistrationResponse
+
+final case class RegistrationTrustResponse(trn : String) extends RegistrationResponse
+
+object RegistrationResponse {
```

Comment:
Please keep the formatting of pattern match consistent.
Either use `{}` or not.
Always return on a new line or not.

Reply:
Done